ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -385,6 +385,7 @@ tests:
       TEST_SUITE: full
     workflow: baremetalds-assisted
 - as: e2e-metal-single-node-live-iso
+  optional: true
```

Comment:
why's that change?

Reply:
a wrong copy paste. 
Please don't review this PR yet, it isn't ready